Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -1,7 +1,14 @@
 { Expression } = require './expression'
+{ FunctionRef } = require './reusable'
+{ decimalAdjust } = require '../util/math'
 { ValueSet, Code } = require '../datatypes/datatypes'
+{ Exception } = require '../datatypes/exception'
 { build } = require './builder'
+ucum = require  'ucum'
 
+module.exports.IncompatibleTypesException = IncompatibleTypesException = class IncompatibleTypesException extends Exception
+  constructor: (@a , @b , e) ->
+    super("Incompatible Types " + @a + " " + @b, e)
```

Comment:
I also suggest changing the error message to:

```
super("Incompatible types '" + @a + "' and  '" + @b + "'", e)
```

So it will look like: "Incompatible types 'wks' and 'wks'"